include/mg_procedure.h

```diff
@@ -1581,6 +1584,10 @@ enum mgp_error mgp_func_result_set_error_msg(struct mgp_func_result *result, con
 /// mgp_func_result.
 enum mgp_error mgp_func_result_set_value(struct mgp_func_result *result, struct mgp_value *value,
                                          struct mgp_memory *memory);
+
+// Log a message on certain level with spdlog
+enum mgp_error mgp_log(const enum mgp_log_level log_level, const char *output);
```

Comment:
```suggestion
enum mgp_error mgp_log(enum mgp_log_level log_level, const char *output);
```

As `log_level` is passed by value, the `const` specifier is a noise for the caller and only important in the definition of the function, not in the declaration.

Reply:
Didn't know that, thanks for the explanation!